src/app/base/components/node/DeleteForm/DeleteForm.test.tsx

```diff
@@ -1,53 +1,48 @@
-import { mount } from "enzyme";
-import { Provider } from "react-redux";
-import { MemoryRouter, Router } from "react-router-dom";
-import { CompatRouter } from "react-router-dom-v5-compat";
 import configureStore from "redux-mock-store";
 
 import DeleteForm from "./DeleteForm";
 
 import type { RootState } from "app/store/root/types";
+import { machine as machineFactory } from "testing/factories";
 import {
-  machine as machineFactory,
-  rootState as rootStateFactory,
-} from "testing/factories";
-import { submitFormikForm } from "testing/utils";
+  getTestState,
+  renderWithBrowserRouter,
+  screen,
+  userEvent,
+} from "testing/utils";
 
-const mockStore = configureStore();
+const mockStore = configureStore<RootState>();
 
 describe("DeleteForm", () => {
   let state: RootState;
 
   beforeEach(() => {
-    state = rootStateFactory();
+    state = getTestState();
```

Comment:
Sorry, having thought about this more I think it makes more sense to make this change it in bulk. It'll be easier as it can be automated. I'll update the epic description and add a separate task for it. 
```suggestion
    state = rootStateFactory();
```